docs/usage/faq.md

```diff
@@ -0,0 +1,49 @@
+# Frequently asked questions
+
+## Why should I use tuist?
+
+When Xcode projects and teams grow, the maintenance becomes cumbersome, and developers spend quite a lot of time fighting issues. Some example of those issues are compilation failing because frameworks are not found, the linker complaining because the framework we are linking against is incompatible. tuist helps you overcome most of those issues by abstracting you away from certain knowledge that Xcode expects your developers to have. With tuist, **your developers can be focused on building the app, rather than setting up the environment and project properly to get Xcode happy about it**.
+
+## What if tuist is not supported in the future?
+
+Not supporting tuist in the future is as likely as Facebook not supporting React. Many things can happen and we can't make sure tuist exists forever. For that reason, we have designed tuist API in such way that your projects **depend weakly on it**. Your team is responsible for defining how the project should look, and tuist takes that input, and generates an Xcode project that follows standard conventions. In case of tuist not being maintained anymore, the generated project is a valid and standard project, which you can include in the git repository.
```

Comment:
Facebook have a nasty habit of dropping support for their tooling. 

also, it would be really good to highlight this being an open source community driven tool.